internal/stage_4.go

```diff
@@ -28,11 +30,11 @@ func testAPIVersionErrorCase(stageHarness *test_case_harness.TestCaseHarness) er
 	correlationId := getRandomCorrelationId()
 	apiVersion := getInvalidAPIVersion()
 
-	broker := protocol.NewBroker("localhost:9092")
+	broker := kafka_client.NewClient("localhost:9092")
```

Comment:
```suggestion
	client := kafka_client.NewClient("localhost:9092")
```

Same as above - when renaming variables, don't leave traces of what somethign was previously named. If you were doing this from scratch, this variable would be `client` and not `broker`